diff --git a/app/views/dashboard/_owner_header.html.slim b/app/views/dashboard/_owner_header.html.slim
index 2be00fe0d..a000cfd7f 100644
--- a/app/views/dashboard/_owner_header.html.slim
+++ b/app/views/dashboard/_owner_header.html.slim
@@ -38,26 +38,6 @@
         -unless current_user.all_owner_collections.empty?
         =link_to t('.create_a_collection'), collection_new_path, data: { controller: 'litebox', litebox: { hash: 'create-collection' } }
 
--if !(current_user.account_type == nil)
-  .owner-info
-    -account_type_key = current_user.account_type.downcase.tr(' ', '_')
-    =t('.account_type', type: t(".account_types.#{account_type_key}"))
-    -unless (current_user.start_date == nil)
-      span
-        |&nbsp;
-        =t('.since', date: l(current_user.start_date.to_date))
-    -unless (current_user.paid_date == nil)
-      span
-        |&nbsp;&middot;&nbsp;
-        =t('.current_subscription_expires', date: l(current_user.paid_date.to_date))
-    span
-      |&nbsp;&middot;&nbsp;
-      =t('.total_pages', pages: current_user.page_count)
-    -if (current_user.account_type == 'Trial') && UPGRADE_FORM_LINK
-      span
-        |&nbsp;&nbsp;
-        =link_to t('.upgrade'), UPGRADE_FORM_LINK, class: 'button outline', target: '_blank'
-
 section.owner-counters
   .counter(data-prefix="#{number_with_delimiter @collections.length}") =t('.collection', count: @collections.length)
   .counter(data-prefix="#{number_with_delimiter @works.length}") =t('.work', count: @works.length)
