diff --git a/app/assets/stylesheets/sections/software.scss b/app/assets/stylesheets/sections/software.scss
index 5124d016c..fe9615028 100644
--- a/app/assets/stylesheets/sections/software.scss
+++ b/app/assets/stylesheets/sections/software.scss
@@ -97,30 +97,6 @@
     }
   }
 
-  #pricing {
-    clear: both;
-    margin-top: $gapSize * 3;
-
-    h2 { margin-bottom: $gapSize * 2; }
-    h3 { text-align: center; }
-
-    ul.pricing_features {
-      list-style: none;
-      margin-left: 0;
-
-      li strong {
-        font-weight: bold;
-      }
-    }
-
-    .pricing_grid_cell .pricing_price .pricing_price_billing {
-      display: block;
-      font-size: $fontSize;
-      width: 100%;
-      margin: 13px 0 $gapSize 0;
-    }
-  }
-
   #integrations {
     section {
       width: 33%;
diff --git a/app/views/contact/send_email.html.slim b/app/views/contact/send_email.html.slim
index c4e737e58..e4fc33b71 100644
--- a/app/views/contact/send_email.html.slim
+++ b/app/views/contact/send_email.html.slim
@@ -6,9 +6,5 @@
 
     p= t('.will_follow_up')
 
-    p= t('.get_started_right_away')
-
-    =link_to t('.start_free_trial'), users_new_trial_path
-
 -unless MIXPANEL_ID.blank?
   javascript: mixpanel.track("Contact Us - Thank You");
diff --git a/app/views/devise/sessions/new.html.slim b/app/views/devise/sessions/new.html.slim
index 103608453..abfd8664a 100644
--- a/app/views/devise/sessions/new.html.slim
+++ b/app/views/devise/sessions/new.html.slim
@@ -38,12 +38,6 @@ section.signon
 
     = link_to t('.sign_up_now'), new_user_registration_path, class: 'button big'
 
-  section#free_trial.signup-links
-    h2= t('.start_free_trial')
-    p= t('.start_free_trial_message')
-
-    =link_to t('.start_free_trial'), users_new_trial_path, class: 'button big'
-
   hr.clearfix
 
 -unless MIXPANEL_ID.blank?
diff --git a/app/views/static/landing_page.html.erb b/app/views/static/landing_page.html.erb
index b1542bd3b..8c6f15158 100644
--- a/app/views/static/landing_page.html.erb
+++ b/app/views/static/landing_page.html.erb
@@ -17,14 +17,6 @@
                             <h4>Start engaging the public to transcribe your documents</h4>
 
                             <div class="bottom">
-                                <p>Upload 200 pages FREE</p>
-                                <% if signed_in?
-                                    target_path = dashboard_role_path
-                                   else
-                                    target_path = users_new_trial_path
-                                   end %>
-
-                                <%= link_to 'Transcribe My Documents', target_path, class: 'button' %>
                             </div>
                         </div>
                     </div>
@@ -601,10 +593,6 @@
             <div class="text">
                 <h4>Start engaging the public to transcribe your documents</h4>
 
-                <div class="buttons">
-                    <a href="https://calendly.com/fromthepage/30-minute-meeting/" target="_blank">Schedule a Demo</a>
-                    <%= link_to 'Upload 200 pages FREE', users_new_trial_path, class: 'second' %>
-                </div>
             </div>
         </div>
     </section>
diff --git a/app/views/static/pricing.html.slim b/app/views/static/pricing.html.slim
deleted file mode 100644
index 6d3b1523e..000000000
--- a/app/views/static/pricing.html.slim
+++ /dev/null
@@ -1,105 +0,0 @@
--content_for :page_title, "Pricing"
--content_for :meta_description, "Pricing for FromThePage collaborative transcription for libraries, archives and museums."
-
-#software
-  h2 Transcribers are always Free!
-
-  p FromThePage connects volunteers transcribers to institutions with material needing transcription.  Institutions pay to host projects on FromThePage, but transcribers -- who are giving their time freely -- are always free.
-
-
-  section#pricing
-    h2 FromThePage Pricing for Organizations of All Sizes
-    p Our pricing is based on how much you expect to use, both in terms of distinct projects and number of page images.
-    .pricing
-      .pricing_grid
-        .pricing_grid_row
-          .pricing_grid_cell
-            h4.pricing_title Researcher
-            ul.pricing_features
-              li.pricing_feature Collaborative transcription, translation, OCR correction and subject indexing
-              li.pricing_feature Multiple export formats (TEI, CSV, HTML, IIIF/Open Annotation)
-              li.pricing_feature 1 project
-              li.pricing_feature Unlimited documents
-              li.pricing_feature Up to 10,000 pages
-              li.pricing_feature E-mail support
-              li.pricing_feature Your data will be available as read-only and can be exported if you cancel your subscription.
-              li.pricing_feature Credit Card Required
-          .pricing_grid_cell
-            h4.pricing_title Small Organization
-            ul.pricing_features
-              li.pricing_feature Collaborative transcription, translation, OCR correction and subject indexing
-              li.pricing_feature Multiple export formats (TEI, CSV, HTML, IIIF/Open Annotation)
-              li.pricing_feature Unlimited projects
-              li.pricing_feature Unlimited documents
-              li.pricing_feature Up to 50,000 pages
-              li.pricing_feature E-mail and chat support
-              li.pricing_feature Phone consultation on new projects
-              li.pricing_feature Your data will be available as read-only and can be exported if you cancel your subscription.
-          .pricing_grid_cell
-            h4.pricing_title Large Institution
-            ul.pricing_features
-              li.pricing_feature Collaborative transcription, translation, subject indexing, and OCR correction
-              li.pricing_feature Multiple export formats (TEI, CSV, HTML, IIIF/Open Annotation)
-              li.pricing_feature Unlimited projects
-              li.pricing_feature Unlimited documents
-              li.pricing_feature Up to 100,000 pages
-              li.pricing_feature Contact us for pricing for additional pages
-              li.pricing_feature E-mail and chat support
-              li.pricing_feature Phone consultation on new projects and strategic consultation
-              li.pricing_feature Your data will be available as read-only and can be exported if you cancel your subscription.
-              li.pricing_feature
-                strong PLUS
-              li.pricing_feature Single Sign On
-              li.pricing_feature On Demand Custom Reports
-	            li.pricing_feature IIIF Consulting
-              li.pricing_feature Integration Consulting
-        .pricing_grid_row
-          .pricing_grid_cell
-            .pricing_price
-              span.pricing_price_value $100
-              .pricing_price_billing per month, billed monthly or 
-              span.pricing_price_value $1200
-              .pricing_price_billing per year, billed annually
-          .pricing_grid_cell
-            .pricing_price
-              span.pricing_price_value $360
-              .pricing_price_billing per month, billed monthly or 
-              span.pricing_price_value $3600
-              .pricing_price_billing per year, billed annually
-          .pricing_grid_cell
-            .pricing_price
-              span.pricing_price_value $600
-              .pricing_price_billing per month, billed monthly or 
-              span.pricing_price_value $6000
-              .pricing_price_billing per year, billed annually
-
-        .pricing_grid_row
-          .pricing_grid_cell
-            =link_to 'Start 200 Page Free Trial', users_new_trial_path, class: 'button big'
-          .pricing_grid_cell
-            =link_to 'Start 200 Page Free Trial', users_new_trial_path, class: 'button big'
-          .pricing_grid_cell
-            =link_to 'Start 200 Page Free Trial', users_new_trial_path, class: 'button big'
-
-  h4 Ready to buy?  
-  p #{link_to('Send us a note', contact_path(contact_form_token))} and we'll send you an invoice, payable either by credit card or by your organization's accounting department.  Need additional paperwork for a PO?  Just let us know.
-
-  h3 What's a page?
-
-  p Every image you upload for transcription is a page.
-
-  h3 What's a project?
-
-  p Projects are distinct in that they are usually the same archival or research collection with the same transcription conventions and other settings (public/private for example).  "Distinct projects" have different transcription conventions and configurations.  For instance, you couldn't do a ledger book and a set of letters in the same project because the configuration is very different for structured data and more textual documents.
-
-  p Sometimes we'll use the words "project" and "collection" interchangeably.
-
-  h3 What happens when we're done transcribing?
-
-  p Page counts are for active projects; once a project is completed you can mark it as inactive and it won't count against your page totals.  Inactive projects are "read only" -- you can view and export the transcriptions, but you can't change them.
-
-  h3 So which one should I choose?
-
-  p The small organization plan is the most popular; most universities fall under it.  State archives usually fit into the large organization plan.  An estimate of the number of pages you'll be working on and how diverse your collections are is the best way to tell.
-
-  h5 Confused?  #{link_to('Drop us a line', contact_path(contact_form_token))} and we'll help you find the best plan for your organization.
diff --git a/app/views/static/signup.html.erb b/app/views/static/signup.html.erb
index b6b5cf907..65f2dfb01 100644
--- a/app/views/static/signup.html.erb
+++ b/app/views/static/signup.html.erb
@@ -14,7 +14,7 @@
                             <h4>I want to upload documents to transcribe.</h4>
         
                             <div class="bottom">
-                                <%= link_to 'Create an Organization Account', users_new_trial_path, class: 'button' %>
+                                Please see the homepage.
                             </div>
                         </div>
                     </div>
diff --git a/app/views/static/software.html.slim b/app/views/static/software.html.slim
index 15c5f4725..67a8b396b 100644
--- a/app/views/static/software.html.slim
+++ b/app/views/static/software.html.slim
@@ -47,83 +47,6 @@
     #video_wrapper
       <iframe title="Get Started" width="560" height="315" src="https://www.youtube.com/embed/UcNXSY0q9uE?controls=0" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
 
-  section#pricing
-    h2 FromThePage Pricing
-
-    .pricing
-      .pricing_grid
-        .pricing_grid_row
-          .pricing_grid_cell
-            h4.pricing_title Researcher
-            ul.pricing_features
-              li.pricing_feature Collaborative transcription, translation, OCR correction and subject indexing
-              li.pricing_feature Multiple export formats (TEI, CSV, HTML, IIIF/Open Annotation)
-              li.pricing_feature 1 project
-              li.pricing_feature Unlimited documents
-              li.pricing_feature Up to 10,000 pages
-              li.pricing_feature E-mail support
-              li.pricing_feature Your data will be available as read-only and can be exported if you cancel your subscription.
-              li.pricing_feature Credit Card Required
-          .pricing_grid_cell
-            h4.pricing_title Small Organization
-            ul.pricing_features
-              li.pricing_feature Collaborative transcription, translation, OCR correction and subject indexing
-              li.pricing_feature Multiple export formats (TEI, CSV, HTML, IIIF/Open Annotation)
-              li.pricing_feature Unlimited projects
-              li.pricing_feature Unlimited documents
-              li.pricing_feature Up to 50,000 pages
-              li.pricing_feature E-mail and chat support
-              li.pricing_feature Phone consultation on new projects
-              li.pricing_feature Your data will be available as read-only and can be exported if you cancel your subscription.
-          .pricing_grid_cell
-            h4.pricing_title Large Institution
-            ul.pricing_features
-              li.pricing_feature Collaborative transcription, translation, subject indexing, and OCR correction
-              li.pricing_feature Multiple export formats (TEI, CSV, HTML, IIIF/Open Annotation)
-              li.pricing_feature Unlimited projects
-              li.pricing_feature Unlimited documents
-              li.pricing_feature Up to 100,000 pages
-              li.pricing_feature Contact us for pricing for additional pages
-              li.pricing_feature E-mail and chat support
-              li.pricing_feature Phone consultation on new projects and strategic consultation
-              li.pricing_feature Your data will be available as read-only and can be exported if you cancel your subscription.
-              li.pricing_feature
-                strong PLUS
-              li.pricing_feature Single Sign On
-              li.pricing_feature On Demand Custom Reports
-              li.pricing_feature IIIF Consulting
-              li.pricing_feature Integration Consulting
-        .pricing_grid_row
-          .pricing_grid_cell
-            .pricing_price
-              span.pricing_price_value $100
-              .pricing_price_billing per month, billed monthly or
-              span.pricing_price_value $1200
-              .pricing_price_billing per year, billed annually
-          .pricing_grid_cell
-            .pricing_price
-              span.pricing_price_value $360
-              .pricing_price_billing per month, billed monthly or 
-              span.pricing_price_value $3600
-              .pricing_price_billing per year, billed annually
-          .pricing_grid_cell
-            .pricing_price
-              span.pricing_price_value $600
-              .pricing_price_billing per month, billed monthly or 
-              span.pricing_price_value $6000
-              .pricing_price_billing per year, billed annually
-
-        .pricing_grid_row
-          .pricing_grid_cell
-            =link_to 'Start Free Trial', users_new_trial_path, class: 'button big'
-          .pricing_grid_cell
-            =link_to 'Start Free Trial', users_new_trial_path, class: 'button big'
-          .pricing_grid_cell
-            =link_to 'Start Free Trial', users_new_trial_path, class: 'button big'
-
-    h3 Ready to buy?  
-    p #{link_to('Send us a note', contact_path(contact_form_token))} and we'll send you an invoice, payable either by credit card or by your organization's accounting department.  Need additional paperwork for a PO?  Just let us know.
-
   #integrations
     h2 Integrations
 
